libs/blocks/section-metadata/section-metadata.js

```diff
@@ -71,6 +77,38 @@ export const getMetadata = (el) => [...el.childNodes].reduce((rdx, row) => {
   return rdx;
 }, {});
 
+async function handleShowMoreButton(section) {
+  const showMoreBtn = section.querySelector('.show-more-button button');
+  if (!showMoreBtn) return;
+  const iconSpan = createTag('span', {
+    class: 'show-more-icon',
+    'aria-hidden': 'true',
+  }, `${ADD_MORE_ICON}`);
+
+  showMoreBtn.innerHTML = `${await replacePlaceholder('see-more-features')}`;
```

Comment:
Kodiak will mark this as a security issue that could easily be avoided by attaching the placeholder where the button is created in `handleCollapseSection`.